tests/SCF/test_SCF.py

```diff
@@ -1,19 +1,70 @@
 import pytest
 import SCF
+from pyscf import gto  # PySCF is a quantum chemistry python module
+import numpy as np
 
 
-def test_calc_nuclear_repulsion_energy(mol_h2o):
-    assert True
-#    assert SCF.calc_nuclear_repulsion_energy(mol_h2o) == 8.00236706181077,\
-#        "Nuclear Repulsion Energy Test (H2O) Failed"
+def test_SCF():

```

Comment:
This does not conform to the assignment, you should have put each test in a separate function and utilized the fixture to run each function.